packages/esm-patient-chart-app/src/ui-components/action-menu.component.tsx

```diff
@@ -21,15 +22,19 @@ export const CHARTS_ACTION_MENU_ITEMS_SLOT = 'action-menu-items-slot';
 export const ActionMenu: React.FC<ActionMenuInterface> = ({ open }) => {
   const { t } = useTranslation();
   const layout = useLayoutType();
-  const { screenMode } = useWorkspace();
+  const { screenMode, active } = useWorkspace();
   const { openWindows, updateWindowSize, windowSize } = useContextWorkspace();
 
   const checkViewMode = () => {
-    if (windowSize.size === 'maximize') {
-      updateWindowSize('hide');
-    } else if (windowSize.size === 'normal') {
-      updateWindowSize('hide');
-    } else updateWindowSize(screenMode ?? 'reopen');
+    if (active) {
+      if (windowSize.size === 'maximize') {
+        updateWindowSize('hide');
+      } else if (windowSize.size === 'normal') {
+        updateWindowSize('hide');
+      } else {
+        updateWindowSize((screenMode as ScreenModeTypes) ?? 'reopen');
```

Comment:
Can `useWorkspace` be changed to return `ScreenModeTypes` for `screenMode`?

Reply:
I have tried but i'm getting a Type error.. screenMode is configured to return a string

Reply:
Well, either that part is of type `ScreenModeTypes` or you change it consistently to be. Otherwise, the typing does not make much sense.